config/initializers/aws.rb

```diff
@@ -0,0 +1,10 @@
+require 'aws-sdk-s3'
+
+if Rails.env.production?
+  Aws.config.update({
+                      region: ENV.fetch('AWS_REGION', nil),
+                      credentials: Aws::Credentials.new(ENV.fetch('AWS_ACCESS_KEY_ID', nil), ENV.fetch('AWS_SECRET_ACCESS_KEY', nil))
+                    })
+
+  S3_BUCKET = Aws::S3::Resource.new.bucket(ENV.fetch('S3_BUCKET', nil))
```

Comment:
is this thread safe?

I wouldn't be confident unless it was specifically said to be as such I would be included to have a method to create a new bucket object when I needed it.